flytekit/tools/script_mode.py

```diff
@@ -142,7 +141,7 @@ def _find_project_root(source_path) -> Path:
     Traverse from current working directory until it can no longer find __init__.py files
     """
     # Start from the directory right above source_path
-    path = Path(source_path).parents[0]
+    path = Path(source_path).parents[0].resolve()
```

Comment:
what does the resolve() do?

Reply:
it turns a relative path into an absolute one. This particular one is useful to find the correct path of the script relative to the project root.